README.md

```diff
@@ -251,6 +253,86 @@ capability might be supported but not enabled.
 | JSCVT     | Perform Conversion to Match Javascript
 | DCPOP     | Persistent Memory Support
 
+### Custom Features
+The Custom feature source allows the user to define features based on a mix of predefined rules.
+
+#### Rule Types
+
+##### PciId Rule
+The PciId rule allows matching the PCI devices in the system on the following PCI attributes: `class`,`vendor` and
+`device` identifiers.
+
+```yaml
+pciId :
+  class: ["<PCI_CLASS_ID>", ...]
+  vendor: ["<PCI_VENDOR_ID>", ...]
+  device: ["<PCI_DEVICE_ID>", ...]
+```
+Matching is done by performing a logical OR for each attribute and logical AND between the specified attributes for
+each PCI device in the system.
```

Comment:
> logical OR for each attribute and logical AND between the specified attributes

A bit cryptic. One suggestion (minimal change): "logical OR for elements of an attribute and logical AND between the specified attributes"